fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/OcflPersistentStorageSessionTest.java

```diff
@@ -153,9 +149,14 @@ public void setUp() throws Exception {
         final var repoDir = tempFolder.newFolder("ocfl-repo").toPath();
         final var workDir = tempFolder.newFolder("ocfl-work").toPath();
 
+        final var objectMapper = new ObjectMapper()
```

Comment:
It might make sense to add a utility method somewhere to get this configuration of an object mapper since it seems to be standard for our `DefaultOcflObjectSessionFactory`s

Reply:
done